README.md

```diff
@@ -108,7 +124,7 @@ with:
   model: 'mistral-large-latest'  # Optional, default model
 ```
 
-### 4. Customize Review Focus (Optional)
+### 5. Customize Review Focus (Optional)
```

Comment:
✨ Nice addition of the customize review focus section! This allows users to tailor the review process.